AnomalyDetection/LeftDiscords/core/src/main/java/software/amazon/com/MPStreamingJob.java

```diff
@@ -0,0 +1,59 @@
+package software.amazon.com;
+
+import org.apache.flink.api.common.serialization.SimpleStringSchema;
+import org.apache.flink.connector.kinesis.sink.KinesisStreamsSink;
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.streaming.connectors.kinesis.FlinkKinesisConsumer;
+import org.apache.flink.streaming.connectors.kinesis.config.AWSConfigConstants;
+import org.apache.flink.streaming.connectors.kinesis.config.ConsumerConfigConstants;
+
+import java.util.Properties;
+
+public class MPStreamingJob {
+
+    private static final String region = "us-east-1";
+    private static final String inputStreamName = "InputStream";
+    private static final String outputStreamName = "OutputStream";
+
+    private static DataStream<String> setSource(StreamExecutionEnvironment env) {
+        Properties inputProperties = new Properties();
+        inputProperties.setProperty(ConsumerConfigConstants.AWS_REGION, region);
+        inputProperties.setProperty(ConsumerConfigConstants.STREAM_INITIAL_POSITION, "LATEST");
+
+        return env.addSource(new FlinkKinesisConsumer<>(inputStreamName, new SimpleStringSchema(), inputProperties));
+    }
+
+    private static KinesisStreamsSink<String> getSink() {
+        Properties outputProperties = new Properties();
+        outputProperties.setProperty(AWSConfigConstants.AWS_REGION, region);
+
+        return KinesisStreamsSink.<String>builder()
+                .setKinesisClientProperties(outputProperties)
+                .setSerializationSchema(new SimpleStringSchema())
+                .setStreamName(outputStreamName)
+                .setPartitionKeyGenerator(element -> String.valueOf(element.hashCode()))
+                .build();
+    }
+
+    public void execute() throws Exception {
+        final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+
+        env.disableOperatorChaining();
```

Comment:
Why? This can impact performance, if this is not explicitly required please remove